src/Layout.jsx

```diff
@@ -24,5 +26,8 @@ const Layout = () => {
     </>
   );
 };
-
+Layout.propTypes = {
```

Comment:
propTypes potentially deprecated in future; 
https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-proptypes-and-defaultprops